src/Extension/FluentSiteTreeExtension.php

```diff
@@ -264,15 +264,14 @@ protected function addLocalePrefixToUrlSegment(FieldList $fields)
                 if ($domain) {
                     $parentBase = Controller::join_links($domain->Link(), Director::baseURL());
                 } else {
-                    $parentBase = Director::absoluteBaseURL();
+                    // Use any existing pre-configured base URL prefix for the field
+                    $parentBase = $segmentField->getURLPrefix();
```

Comment:
Ah, the bug is that this link includes the parent url, whereas so does $parentRelative. It's adding parent urlsegment twice.

Reply:
I have to revert this at https://github.com/tractorcow-farm/silverstripe-fluent/pull/539